### PR TITLE
[ci skip-rerun] Escape underscores in `LIKE` patterns

### PR DESCRIPTION
**Pull request by sean-rose**
In `LIKE` patterns unescaped underscores will match any single character.

### CODE REVIEW

File: jetstream/bookmarks-toolbar-default-on.toml [hunk 1]
```diff
@@ -5,12 +5,12 @@ overall = ['bookmarks_bar_pinned_on', 'bookmarks_bar_pinned_off', 'bookmarks_bar
 
 
 [metrics.bookmarks_bar_pinned_on]
-select_expression = "CAST(COALESCE(SUM(CASE WHEN(SELECT CAST(COUNT(t) AS BOOL) FROM UNNEST(payload.processes.parent.keyed_scalars.browser_ui_toolbar_widgets) t WHERE t.key LIKE 'bookmarks-bar_pinned_on' AND t.value) THEN 1 ELSE 0 END),0) > 0 AS INT)"
+select_expression = '''CAST(COALESCE(SUM(CASE WHEN(SELECT CAST(COUNT(t) AS BOOL) FROM UNNEST(payload.processes.parent.keyed_scalars.browser_ui_toolbar_widgets) t WHERE t.key LIKE r'bookmarks-bar\_pinned\_on' AND t.value) THEN 1 ELSE 0 END),0) > 0 AS INT)'''
```
Comment (sean-rose) [inline jetstream/bookmarks-toolbar-default-on.toml:8]: This `LIKE` pattern doesn't have any `%` wildcards, so I suppose the underscores could have been meant as wildcards, but I think it's unlikely given the context.

File: jetstream/bookmarks-toolbar-default-on.toml [hunk 2]
```diff
@@ -5,12 +5,12 @@ overall = ['bookmarks_bar_pinned_on', 'bookmarks_bar_pinned_off', 'bookmarks_bar
...
 data_source = 'main'
 
 
 [metrics.bookmarks_bar_pinned_off]
-select_expression = "CAST(COALESCE(SUM(CASE WHEN(SELECT CAST(COUNT(t) AS BOOL) FROM UNNEST(payload.processes.parent.keyed_scalars.browser_ui_toolbar_widgets) t WHERE t.key LIKE 'bookmarks-bar_pinned_off' AND t.value) THEN 1 ELSE 0 END),0) > 0 AS INT)"
+select_expression = '''CAST(COALESCE(SUM(CASE WHEN(SELECT CAST(COUNT(t) AS BOOL) FROM UNNEST(payload.processes.parent.keyed_scalars.browser_ui_toolbar_widgets) t WHERE t.key LIKE r'bookmarks-bar\_pinned\_off' AND t.value) THEN 1 ELSE 0 END),0) > 0 AS INT)'''
```
Comment (sean-rose) [inline jetstream/bookmarks-toolbar-default-on.toml:13]: ditto

File: opmon/desktop-dau.toml [hunk 3]
```diff
@@ -59,9 +59,9 @@ from_expression = """
         FROM `moz-fx-data-shared-prod.telemetry_derived.kpi_forecasts_v0` AS forecasts
         JOIN most_recent_forecasts
        USING(aggregation_period, metric_alias, metric_hub_app_name, metric_hub_slug, forecast_predicted_at)
-       WHERE aggregation_period = 'day' AND measure IN ('observed', 'p50') AND metric_alias LIKE 'desktop_dau'
+       WHERE aggregation_period = 'day' AND measure IN ('observed', 'p50') AND metric_alias LIKE r'desktop\_dau'
```
Comment (sean-rose) [inline opmon/desktop-dau.toml:62]: ditto

**Comment by sean-rose** [comment]
All the [validate-jetstream CI errors](https://app.circleci.com/pipelines/github/mozilla/metric-hub/1360/workflows/04ba8832-3ff5-4d61-aa4e-fc7867ccdb83/jobs/2734) seem to be due to metrics referencing `main_v4` and getting permission denied, which I think is expected at this point.

**Review by scholtzan — APPROVED** [review]

**Pull request merged**